### PR TITLE
Categories fixes

### PR DESCRIPTION
**Pull request by tyom**

### CODE REVIEW

**Comment by tyom** [comment]
test this please

File: cla_public/apps/checker/views.py [hunk 1]
```diff
@@ -363,7 +363,7 @@ def get_context(self, category_name):
             name = unicode(name)
 
             category_name = ORGANISATION_CATEGORY_MAPPING.get(name, name)
-        trans_category_name = ORGANISATION_CATEGORY_MAPPING.get(name, name)
+        trans_category_name = next(c[1] for c in CATEGORIES if c[0] == category)
```
Comment (ushkarev) [inline cla_public/apps/checker/views.py]: could `category` in this line ever be missing from `CATEGORIES`? if so, then this would throw an error

**Comment by ushkarev** [comment]
:+1:

File: cla_public/apps/checker/views.py (hunk 1 above)
Comment (tyom) [inline cla_public/apps/checker/views.py]: Lines 358-361 should cover that case.

**Comment by tyom** [comment]
test this please

**Pull request merged**